File: problem.0010/problem10.py
#!python3

max = 2000000
n = 3
s = set(range(2,max+1))
p = {2,3}

while n < max+1 :

        s = s - set(range(n,max+1,n))

        if not s :
            break
        else :
            n = min(s)

        p.add(n)
        #print(n)

print(sum(p))

File: problem.0010/problem10.pl
#!/usr/bin/perl
use strict;

use lib '../lib';
use Euler::Prime::Modulus '+cache';

my $MAX = 2000000;

my @primes = ();

for( my $i = 2; $i < $MAX; $i++ ){
	push( @primes, $i ) if is_prime( $i );	
}

my $sum = 0;
$sum += $_ for @primes;

print "$sum\n";
